File: projects/myClass/character.py
class Character:
    def __init__(self, dead : bool = False):
        self._dead : bool = dead

    @property
    def Dead(self) -> bool:
        return self._dead

    def SetAlive(self) -> None:
        self._dead = False

    def SetDead(self) -> None:
        self._dead = True

    @property
    def Killed(self) -> bool:
        return self._dead

File: projects/c++class2/myClassC++/src/character.cpp
#include "character.h"

namespace go {

    Character::Character(bool _Dead)
      :  Dead(_Dead)
    {
    }
    void Character::setdead() {
        Dead = true;
    }
    void Character::setalive() {
        Dead = false;
    }

    bool Character::isDead() const {
        return Dead;
    }

 }

